Desktop/data/datasettablemodel.cpp

```diff
@@ -21,6 +21,9 @@
 DataSetTableModel::DataSetTableModel() : DataSetTableProxy(parIdxType::data)
 {
 	connect(DataSetPackage::pkg(),	&DataSetPackage::columnsFilteredCountChanged,	this, &DataSetTableModel::columnsFilteredCountChanged	);
+	connect(DataSetPackage::pkg(),	&DataSetPackage::columnDataTypeChanged,			this, &DataSetTableModel::columnTypeChanged				);
+	connect(DataSetPackage::pkg(),	&DataSetPackage::labelChanged,					this, &DataSetTableModel::labelChanged					);
```

Comment:
I assume this signal is sent in addition to `dataChanged` and not as a replacement?

Reply:
yes